### PR TITLE
Feature/admin wizard

### PR DESCRIPTION
**Pull request by annashamray**
Fixes # https://github.com/open-zaak/open-zaak/issues/542

**Changes**
Move step 1 and step 2 of config wizard to the admin

### CODE REVIEW

File: src/openzaak/config/templates/admin/config_detail.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                <caption>NLX</caption>
+                <thead>
+                    <tr>
+                        <th scope="col">Parameter</th>
+                        <th scope="col">Waarde</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><strong>NLX directory</strong></td>
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_detail.html:28]:
```suggestion
                        <td><strong>{% trans "NLX directory" %}</strong></td>
```

File: src/openzaak/config/templates/admin/config_detail.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                        <td>{{ nlx.directory_url| urlize }}</td>
+                    </tr>
+                    <tr>
+                        <td><strong>Outway address</strong></td>
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_detail.html:32]:
```suggestion
                        <td><strong>{% trans "NLX outway address" %}</strong></td>
```

File: src/openzaak/config/templates/admin/config_detail.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                        <th scope="col">Service</th>
+                        <th scope="col" title="Service geactiveerd en aangeboden door Open Zaak">
+                            Geactiveerd
+                        </th>
+                        <th scope="col" title="Service wordt aangeboden over NLX">
+                            NLX
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_detail.html:54]: More `{ % trans %}`

File: src/openzaak/config/templates/admin/config_detail.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        <section class="module">
+            <table class="table">
+                <caption>Open Zaak services</caption>
+                <thead>
+                    <tr>
+                        <th scope="col">Service</th>
+                        <th scope="col" title="Service geactiveerd en aangeboden door Open Zaak">
+                            Geactiveerd
+                        </th>
+                        <th scope="col" title="Service wordt aangeboden over NLX">
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_detail.html:49]:
```suggestion
                        <th scope="col" title="Service wordt ook aangeboden over NLX">
```

File: src/openzaak/config/templates/admin/config_detail.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                            NLX
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_detail.html:50]:
```suggestion
                            Aanbieden via NLX
```

File: src/openzaak/config/templates/admin/config_detail.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+                            {{ service.nlx|boolean_icon }}
+                        </td>
+                    </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        </section>
+
+        <div class="submit-row">
+            <input type="button" class="default" value="{% trans 'Wijzigen' %}" onclick="location.href='{% url 'config:config-nlx' %}';">
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_detail.html:71]: Rename button to "Wizard configuratie" and add query param to add wizard-like features (see other comments)

```suggestion
            <input type="button" class="default" value="{% trans 'Wizard configuratie' %}" onclick="location.href='{% url 'config:config-nlx' %}?wizard=true';">
```

File: src/openzaak/config/templates/admin/config_internal.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; <a href="{% url 'config:config-detail' %}">{% trans 'Service configuratie' %}</a>
+&rsaquo; <a href="{% url 'config:config-nlx' %}">{% trans 'NLX' %}</a>
+&rsaquo; {% trans 'Open Zaak services' %}
+</div>
+{% endblock %}
+
+{% block content %}
+<h1>{% trans 'Service configuratie: Open Zaak' %}</h1>
+<h3>Step 2 of 2</h3>
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_internal.html:19]: Wizard mode:

```suggestion
{% if request.GET.wizard=True %}<h3>Step 2 of 2</h3>
```

File: src/openzaak/config/templates/admin/config_detail.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+                        <td><strong>NLX directory</strong></td>
+                        <td>{{ nlx.directory_url| urlize }}</td>
+                    </tr>
+                    <tr>
+                        <td><strong>Outway address</strong></td>
+                        <td>{{ nlx.outway| urlize }}</td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_detail.html:37]: Add change button "Wijzigen" for this section (non-wizard)

File: src/openzaak/config/templates/admin/config_detail.html [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+                        <td>
+                            {{ service.enabled|boolean_icon }}
+                        </td>
+                        <td>
+                            {{ service.nlx|boolean_icon }}
+                        </td>
+                    </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_detail.html:67]: Add change button "Wijzigen" for this section (non-wizard)

File: src/openzaak/config/templates/admin/config_internal.html [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+                        <td>{{ form.enabled }}</td>
+                        <td>{{ form.nlx }}</td>
+                    </tr>
+                {% endfor %}
+                </tbody>
+            </table>
+        </fieldset>
+
+        <div class="submit-row">
+            <input class="default" type="submit" name="submit" value="{% trans "submit" %}" />
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_internal.html:53]: pseudo code:

```
if wizard:
  show previous/skip/next/done buttons (vorige/volgende/voltooien) if appropriate
else:
  show cancel/save buttons (annuleren/opslaan)
```

File: src/openzaak/config/templates/admin/config_nlx.html [hunk 11]
```diff
@@ -0,0 +1,44 @@
+{% extends "admin/base_site.html" %}
```
Comment (joeribekker) [inline src/openzaak/config/templates/admin/config_nlx.html:1]: same wiz-like comments as `config_internal.html`

File: src/openzaak/config/templates/admin/config_internal.html (hunk 10 above)
Comment (annashamray) [reply src/openzaak/config/templates/admin/config_internal.html:53]: done

File: src/openzaak/config/templates/admin/config_detail.html (hunk 8 above)
Comment (annashamray) [reply src/openzaak/config/templates/admin/config_detail.html:37]: the button is added

File: src/openzaak/config/templates/admin/config_detail.html (hunk 9 above)
Comment (annashamray) [reply src/openzaak/config/templates/admin/config_detail.html:67]: the button is added

File: src/openzaak/config/templates/admin/config_nlx.html (hunk 11 above)
Comment (annashamray) [reply src/openzaak/config/templates/admin/config_nlx.html:1]: done

**Pull request merged**